docs/user-guide/connectors/facebook-messenger.rst

```diff
@@ -69,3 +69,78 @@ The endpoint for receiving Facebook messenger messages is
 ``http://localhost:5005/webhooks/facebook/webhook``, replacing
 the host and port with the appropriate values. This is the URL
 you should add in the configuration of the webhook.
+
+Supported response attachments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In addition to typical text, image, button, and custom responses, the Facebook Messenger channel supports the following additional response template attachments:
```

Comment:
```suggestion
In addition to typical text, image, and custom responses, the Facebook Messenger channel supports the following additional response template attachments:
```
because we also reference how buttons are different here